AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -108,6 +108,8 @@ vayamo.ch##.cookies_overlay_wrapper
 hotukdeals.com##.popover--oreo-message
 canto.com#$#.cc-wrapper { display: none !important; }
 canto.com#$#body.noscroll { overflow: visible !important; padding-right: 0 !important; }
+wordpress.com###cmp-app-container
+wordpress.com##div[class$="-cookie-banner"]
```

Comment:
It seems that this rule doesn't work (at least on my end), because class name doesn't end with `-cookie-banner`.
```
class="a8c-cookie-banner a8c-cookie-banner-v1"
```

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/115253817-09814880-a12d-11eb-8707-9205a47b8bcf.png)

</details>

Reply:
@AdamWr, i can't reproduce it anymore for some reason, but i swapped selector to `.a8c-cookie-banner`